pkg/cmd/certmanager_app.go

```diff
@@ -71,7 +67,7 @@ func makeInstallCertManager() *cobra.Command {
 			}
 		}
 
-		nsRes, nsErr := kubectlTask("create", "namespace", namespace)
+		nsRes, nsErr := kubectl(command, "create", "namespace", namespace).Execute()
```

Comment:
It looks as though we are passing the entire command into this func just to get the kubeconfig flag. Probably more readable to just pass in the flag every time, then people know why it's sent

Reply:
The Idea of passing down the command came to my mind because currently in every app we do this: 
https://github.com/alexellis/k3sup/blob/master/cmd/apps/certmanager_app.go#L29-L35

Which means in order to use the kubectl function we have to repeat 6 - 12 lines in every app install. (12 when we add kubecontext as a flag) Which IMHO does not quite scale well if we add more and more apps and then making a change to the kubectl function every app is affected.